gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -269,6 +280,42 @@
             <property name="format" value="^import org\.apache\.commons\.math\."/>
             <property name="message" value="math is deprecated, use math3 instead"/>
         </module>
+        <module name="RegexpSinglelineJava">
+            <property name="format" value="\bPreconditions\.check.*\{\}"/>
+            <property name="message" value="Use printf-style formatting in Preconditions"/>
+            <property name="ignoreComments" value="true"/>
+        </module>
+        <module name="RegexpSinglelineJava">
+            <property name="format" value="\.(debug|info|warn|error)\(.*%[sd]"/>
```

Comment:
this presumes that slf4j is used. is that a valid assumption? is this check prone to yield false positives?